templates/human.html

```diff
@@ -210,20 +210,20 @@ <h4>Human behavior configuration:</h4>
-                    '&& cd '+humanName+' && virtualenv -p python3 '+humanName+' && '+humanName+'/bin/pip install -r requirements.txt && '+humanName+'/bin/python human.py --clustersize '+taskCount+' ' +
+                baseHuman = 'curl -sk -o \''+humanName+'.tar.gz\' -X POST -H \'file:'+humanName+'.tar.gz\' '+http+'/file/download 2>&1 && mkdir \''+humanName+'\' && tar -C \''+humanName+'\' -zxvf \''+humanName+'.tar.gz\' ' +
+                    '&& cd \''+humanName+'\' && virtualenv -p python3 \''+humanName+'\' && \''+humanName+'/bin/pip\' install -r requirements.txt && \''+humanName+'/bin/python\' human.py --clustersize '+taskCount+' ' +
                     '--taskinterval '+taskInterval+' --taskgroupinterval '+taskClusterInterval+' --extra '+extra;
                 break;
             case "linux":
-                baseHuman = 'curl -sk -o '+humanName+'.tar.gz -X POST -H \'file:'+humanName+'.tar.gz\' '+http+'/file/download 2>&1 && mkdir '+humanName+' && tar -C '+humanName+' -zxvf '+humanName+'.tar.gz ' +
-                    '&& cd '+humanName+' && virtualenv -p python3 '+humanName+' && '+humanName+'/bin/pip install -r requirements.txt && '+humanName+'/bin/python human.py --clustersize '+taskCount+' ' +
+                baseHuman = 'curl -sk -o \''+humanName+'.tar.gz\' -X POST -H \'file:'+humanName+'.tar.gz\' '+http+'/file/download 2>&1 && mkdir \''+humanName+'\' && tar -C \''+humanName+'\' -zxvf \''+humanName+'.tar.gz\' ' +
+                    '&& cd \''+humanName+'\' && virtualenv -p python3 \''+humanName+'\' && \''+humanName+'/bin/pip\' install -r requirements.txt && \''+humanName+'/bin/python\' human.py --clustersize '+taskCount+' ' +
                     '--taskinterval '+taskInterval+' --taskgroupinterval '+taskClusterInterval+' --extra '+extra;
                 break;
             case "windows-psh":
                 baseHuman = '$server="'+http+'"; $url="$server/file/download"; $wc=New-Object System.Net.WebClient; $wc.Headers.add("file","'+humanName+'.zip"); $wc.DownloadFile($url, "$pwd\\'+humanName+'.zip"); ' +
-                    'Expand-Archive '+humanName+'.zip -DestinationPath '+humanName+' -Force; cd '+humanName+'; python.exe -m venv '+humanName+';' +
-                    '.\\'+humanName+'\\Scripts\\pip.exe install -r requirements.txt; .\\'+humanName+'\\Scripts\\python.exe human.py --clustersize '+taskCount+' ' +
-                    '--taskinterval '+taskInterval+' --taskgroupinterval '+taskClusterInterval+' --extra '+extra;
+                    'Expand-Archive "'+humanName+'.zip" -DestinationPath "'+humanName+'" -Force; cd "'+humanName+'"; python.exe -m venv "'+humanName+'";' +
+                    'Start-Process -FilePath ".\\'+humanName+'\\Scripts\\pip.exe" -ArgumentList "install -r requirements.txt"; Start-Process -FilePath ".\\\\'+humanName+'\\\\Scripts\\\\python.exe" -ArgumentList "human.py --clustersize '+taskCount+' ' +
```

Comment:
```suggestion
                    'Start-Process -FilePath ".\\'+humanName+'\\Scripts\\pip.exe" -ArgumentList "install -r requirements.txt" -Wait; Start-Process -FilePath ".\\\\'+humanName+'\\\\Scripts\\\\python.exe" -ArgumentList "human.py --clustersize '+taskCount+' ' +
```
From running a quick test on my end, it seems like the second python.exe process is getting kicked off and not waiting for the pip process to finish. Adding the `-Wait` flag should resolve